config/initializers/apipie.rb

```diff
@@ -27,6 +27,8 @@
     trans
   end
   config.help_layout = 'apipie_dsl/apipie_dsls/help.html.erb'
+  config.default_model_description = ->(model) { _("A class representing %s object") % model.model_name.singular.humanize(capitalize: false) }
```

Comment:
If we can call translations here, then I'd expect this to also work:
```suggestion
  config.default_model_description = ->(model) { _("A class representing %s object") % model.human }
```

Reply:
Test failures seem to be related :/